carsons/carsons.py

```diff
@@ -265,3 +275,39 @@ def GMR_cn(self, phase) -> float:
         k = self.neutral_strand_count[phase]
         R = self.radius[phase]
         return (GMR_s * k * R**(k-1))**(1/k)
+
+
+class MultiConductorCarsonsEquations(CarsonsEquations):
+    def __init__(self, model):
+        super().__init__(model)
+        self.radius: Dict[str, float] = model.radius
+        self.secondary: bool = model.secondary
+        self.insulation_thickness: Dict[str, float] = \
+            model.insulation_thickness
+
+    def compute_d(self, i, j) -> float:
+        return (self.radius[i] + self.radius[j]
+                + self.insulation_thickness[i] + self.insulation_thickness[j])
+
+    def compute_D(self, i, j) -> float:
+        return self.phase_positions[i][1] + self.phase_positions[j][1]
+
+    @property
+    def conductors(self):
+        neutral_conductors = sorted([
+            ph for ph in self.phases
+            if ph.startswith("N")
+        ])
+        if self.secondary:
```

Comment:
wondering if we could use the phase keys to indicate if its a secondary or not?

for example, usually an object would have something like 
```
LineModel(
    positions={
        A: x,y,
        B: x,y, 
        C: x,y, 
        N: x,y
    }
)
``` 

What if we let the user supply stuff using `S1` and `S2` as the keys for everything, e.g. 
```
Multiconductor(positions={
    S1: x,y,
    S2: x,y,
    N: x,y}
)
```

That way the user doesn't need to pass in an is_secondary flag, and there would never be a case where they e.g. set `is_secondary=True` but pass in A/B/C/N data, which would likely lead to a key-error

Reply:
yeah that would work. I wasn't sure if this input format would make sense for the user because everything else is in ABC phases. Maybe it's fine and this is what the user expects?

Reply:
updated in 029e727